test/query_hash_coercer_test.rb

```diff
@@ -0,0 +1,57 @@
+require_relative "test_helper"
+
+describe Committee::QueryHashCoercer do
```

Comment:
Excellent new tests! Thanks :)

Would you mind just adding cases for what happens with each type in case an invalid value is encountered?